client/components/handTrack.jsx

```diff
@@ -0,0 +1,12 @@
+import * as handTrack from 'handtrackjs';
+
+const img = document.getElementById('img');
+
+// loading the model
+handTrack.load().then((model) => {
+  // detect objects in the image
+  console.log('model loaded');
+  model.detect(img).then((predictions) => {
+    console.log('Predictions: ', predictions);
+  });
+});
```

Comment:
Since this is in the components folder, is this going to be converted into a react component?

Reply:
I removed this file since it will be in my next PR for the drawing. But yes, it will be part of a react component in the next PR.